frontend/src/page/Realm.tsx

```diff
@@ -1,26 +1,32 @@
 import React from "react";
-import { Link } from "react-router-dom";
 
 import { graphql, useLazyLoadQuery } from "react-relay/hooks";
 import { RealmQuery } from "../query-types/RealmQuery.graphql";
 
 import { Breadcrumbs } from "../ui/Breadcrumbs";
+import { NavMain as MainLayout } from "../layout/NavMain";
+import { unreachable } from "../util/err";
 
 
 type Props = {
     path: string;
 };
 
-export const Realm: React.FC<Props> = ({ path }) => {
+export const RealmPage: React.FC<Props> = ({ path }) => {
     const isRoot = path === "";
 
     // TODO Build this query from fragments!
     const { realm } = useLazyLoadQuery<RealmQuery>(graphql`
         query RealmQuery($path: String!) {
             realm: realmByPath(path: $path) {
+                id
                 name
+                path
                 parents { name path }
                 children { id name path }
+                parent {
+                    children { id name path }
+                }
```

Comment:
Can't we get this from the `parents` array? Should we? Dunno, just :thinking: out loud.

Reply:
Mhh we don't really want to get the children of all ancestors, right? (Maybe the thing should be called `ancestors` instead of `parents`?)

Reply:
+1 for `ancestors`. And yeah, fair point.